e2e_playwright/custom_components/popular_components_test.py

```diff
@@ -14,13 +14,16 @@
 
 from playwright.sync_api import Page, expect
 
+from e2e_playwright.conftest import wait_for_app_run
+
 
 def _select_component(app: Page, component: str):
     selectbox_input = app.get_by_test_id("stSelectbox").locator("input")
 
     # Type an option (defined in the test app):
     selectbox_input.type(component)
     selectbox_input.press("Enter")
+    wait_for_app_run(app)
```

Comment:
Added this wait as without it the exception we are looking for was usually not detected :/